apps/docs/pages/docs/api-reference/components/puck.mdx

```diff
@@ -182,16 +182,16 @@ Render the Puck preview within iframe. Defaults to `true`.
 
 Disabling iframes will also disable [viewports](#viewports).
 
-### `initialHistories`
+### `initialHistory`
 
-Sets the undo/redo Puck history state.
+Sets the undo/redo Puck history state when using the usePuck [history API](/docs/api-reference/functions/use-puck#history).
```

Comment:
Should usePuck be `backticked` i.e. `usePuck`? Seems like this would be consistent with other code references in docs.